test/integration/aaa_download_only_test.go

```diff
@@ -74,6 +74,7 @@ func TestDownloadOnly(t *testing.T) {
 
 					// skip for none, as none driver does not have preload feature.
 					if !NoneDriver() {
+						viper.Set("preload", true)
```

Comment:
this is not the right way to do this. setting global flag. please pass this as an arg to PreloadExists. maybe as an optional argument
...preload bool

Reply:
is a hidden boolean trying to emulate a global flag really better than just using the flag?